app/commands.py

```diff
@@ -1407,6 +1407,9 @@ async def rmpriv(ctx: Context) -> Optional[str]:
     if not (t := await app.state.sessions.players.from_cache_or_sql(name=ctx.args[0])):
         return "Could not find user."
 
+    if bits & Privileges.DONATOR:
+        return "Please use the !givedonator command to assign donator privileges to players."
```

Comment:
wrong function, this is rmpriv hehehe

if you fix i can merge

Reply:
Lmfao yeah gonna fix in like 9 hours